### DISCUSSION TITLE
Increase the body parser limit in gatsby api

### DISCUSSION THREAD

**Discussion by SukantaB** [gatsbyjs/gatsby#35175] [Help] [answered]
Hi is there any way to increase the body parser limit in Gatsby api functions ?

I am using api function, to send emails but while using attachments i am getting 413 error . anyone has any solution ? mail service is mailgun

**Comment by LekoArts** [comment, marked as answer]
It's a feature that is actively been worked on https://github.com/gatsbyjs/gatsby/discussions/35084
